activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -471,10 +472,10 @@ def cache_dump_filename(db_config, schema_cache_path: nil)
           db_config.default_schema_cache_path(ActiveRecord::Tasks::DatabaseTasks.db_dir)
       end
 
-      def load_schema_current(format = ActiveRecord.schema_format, file = nil, environment = env)
```

Comment:
Changes the signature of a public method. Maybe instead keep the parameter, defaulted to nil, and use it instead of `db_config.schema_format` if it's non-nil?

Reply:
Sounds good.